csharp/src/Microsoft.ML.OnnxRuntime/OnnxRuntime.shared.cs

```diff
@@ -150,6 +151,26 @@ public string[] GetAvailableProviders()
 
             return availableProviders;
         }
+
+
+        /// <summary>
+        /// Get current log level of OrtEnv instance
+        /// </summary>
+        /// <returns>current log level</returns>
+        public LogLevel GetLogLevel()
+        {
+            return currentLogLevel;
+        }
```

Comment:
If this is really needed, it should be a property. And you can then combine set/get

Reply:
Thanks for sharing! This has been improved in the [latest commit ](https://github.com/microsoft/onnxruntime/pull/13402/commits/9715a16b2b77b5a6ff59326c981c433e66b9c0d5)and verified in test case.